dev/tpv2/run.sh

```diff
@@ -16,5 +16,24 @@
 set -o errexit
 
 cd "$TC/experimental/traffic-portal"
+
+user=tpv2
+uid="$(stat -c%u "$TC")"
+gid="$(stat -c%g "$TC")"
+if [[ "$(id -u)" != "$uid" ]]; then
+	for dir in "${TC}/.npm" .angular node_modules; do
```

Comment:
this fails on fresh repo clones:
```
tpv2_1            | stat: cannot statx '.angular': No such file or directory
tpv2_1            | chown: cannot access '.angular': No such file or directory
```
(and then the container crashes)

Reply:
Checking if the directory exists first in ec0651f0e2